test/filter.js

```diff
@@ -42,8 +42,7 @@ describe('Bigtable/Filter', function() {
   });
 
   afterEach(function() {
-    sinon.restore();
```

Comment:
We should probably start using http://sinonjs.org/releases/v4.4.4/sandbox/ on the afterEach since the world won't be in a broken state if an assert fails

Reply:
Hmm, I'm not sure why I removed this line, since it looks like it's still valid. I'll put it back -- does that fix what you were calling out, or is there more to do?

Reply:
I think it's still worth using sandboxes since you need to remember to call restore for each spy. It doesn't need to happen right now, but if a test fails in the future it can cascade to every other test after it

Reply:
Earlier in the file, `sinon` is assigned as:

```js
var sinon = require('sinon').sandbox.create();
```

Maybe I don't need the `FakeMutation.convertToBytes.resetHistory();` in the afterEach now?

Reply:
Whoops missed that bit, reverting this line should be all that's needed then

Reply:
Great, here's the new afterEach:

```js
afterEach(function() {
  sinon.restore();
});
```